cpp/src/aztec/honk/proof_system/verifier.test.cpp

```diff
@@ -114,11 +114,11 @@ template <class FF> class VerifierTests : public testing::Test {
             w_o[2 * i] = w_o[2 * i] + w_l[2 * i];
             w_o[2 * i] = w_o[2 * i] + w_r[2 * i];
             w_o[2 * i] = w_o[2 * i] + fr::one();
-            fr::__copy(fr::one(), q_l.at(2 * i));
-            fr::__copy(fr::one(), q_r.at(2 * i));
-            fr::__copy(fr::neg_one(), q_o.at(2 * i));
-            fr::__copy(fr::one(), q_c.at(2 * i));
-            fr::__copy(fr::one(), q_m.at(2 * i));
+            q_l.at(2 * i) = fr::one();
+            q_r.at(2 * i) = fr::one();
+            q_o.at(2 * i) = fr::neg_one();
+            q_c.at(2 * i) = fr::one();
+            q_m.at(2 * i) = fr::one();
```

Comment:
There is no reason the compiler cannot optimize the default assignment to be as performant as `__copy`. This makes things clearer